CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(CMAKE_VERBOSE_MAKEFILE ON)
 
 option(BUILD_BINDINGS "Build the python bindings" ON)
 option(MorphIO_CXX_WARNINGS "Compile C++ with warnings" ON)
+option(MorphIO_WERROR "If compiled with warnings, additionally activate '-Werror'" OFF)
```

Comment:
That would lead to some double negation logic. We could just switch the default to `On`. Either is fine, as long as I can compile with warnings without them preventing progress by turning into errors.

Reply:
If `*_NO_WERROR` is your preferred name, I'm happy to change it. For now I only updated the default.